_posts/pl/2021-01-11-rest-api-dobre-praktyki.md

```diff
@@ -14,26 +14,26 @@ tags:
 
 
 ## HTTP i REST - czy zawsze dobrze je stosujemy?
-Jakie są częste błędy przy doborze metod HTTP?  Kiedy trzymać się standardu, a kiedy może to być problemem? O czym pamiętać projektując swoje REST API? Choć ani standard HTTP, ani REST nie są nowością w świecie IT, to pewne twierdzenia ich dotyczące są błędnie powielane przez wiele osób. Czasem znów okazuje się, że deweloperzy z powszechnie znanych firm podchodzą do tego samego problemu w inny sposób. Zapraszam na podróż w znane, ale wciąż jeszcze nie zawsze przestrzegane standardy HTTP i REST. 
+Jakie są częste błędy przy doborze metod HTTP?  Kiedy trzymać się standardu, a kiedy może to być problemem? O czym pamiętać projektując swoje REST API? Choć ani standard HTTP, ani REST nie są nowością w świecie IT, to pewne twierdzenia ich dotyczące są błędnie powielane przez wiele osób. Czasem znów okazuje się, że deweloperzy pracujący dla różnych firm podchodzą do tego samego problemu w inny sposób. Zapraszam na podróż w znane, ale wciąż jeszcze nie zawsze przestrzegane, standardy HTTP i REST. 
 
 ## Kiedy POST, a kiedy PUT? 
 
 ### POST do tworzenia, a PUT do modyfikacji? Niekoniecznie
-Często można spotkać się z prostym wyjaśnieniem, kiedy używać metody POST, a kiedy PUT. Kiedy tworzymy nowy zasób — POST, kiedy modyfikujemy istniejący — PUT. Jest to jednak znaczne uproszczenie i pierwszy mit, z którym chcę się rozprawić.
+Często można spotkać się z prostym (lecz błędnym) wyjaśnieniem, kiedy używać metody POST, a kiedy PUT. Kiedy tworzymy nowy zasób — POST, kiedy modyfikujemy istniejący — PUT. Jest to jednak znaczne uproszczenie i pierwszy mit, z którym chcę się rozprawić.
```

Comment:
Twoja poprawka już wygląda dużo lepiej. Jeszcze tylko wątpliwości budzi drugie zdanie w tym akapicie. Czy to są pytania, które stawiasz? Wówczas musiałby je kończyć znak zapytania. A może właśnie odpowiedź do pytania/mitu postawionego w pierwszym zdaniu? Wówczas sugerowałbym jakieś spoiwo w stylu "twierdzi się, że kiedy tworzymy nowy zasób używamy POST, a kiedy modyfikujemy istniejący — PUT

Reply:
@bmitan ? :)

Reply:
Poproszę o sprawdzenie teraz